src/main/java/frc/trigon/robot/subsystems/roller/placeholderroller/PLACEHOLDERRollerIO.java

```diff
@@ -0,0 +1,40 @@
+package frc.trigon.robot.subsystems.roller.placeholderroller;
+
+import com.ctre.phoenix6.StatusSignal;
+import com.ctre.phoenix6.controls.VelocityTorqueCurrentFOC;
+import com.ctre.phoenix6.hardware.TalonFX;
+import edu.wpi.first.wpilibj.DigitalInput;
+import frc.trigon.robot.subsystems.roller.RollerConstants;
+import frc.trigon.robot.subsystems.roller.RollerIO;
+import frc.trigon.robot.subsystems.roller.RollerInputsAutoLogged;
+
+public class PLACEHOLDERRollerIO extends RollerIO {
+    private final TalonFX motor = PLACEHOLDERRollerConstants.MOTOR;
+    private final DigitalInput infraredSensor = PLACEHOLDERRollerConstants.INFRARED_SENSOR;
+    private final VelocityTorqueCurrentFOC currentRequest = new VelocityTorqueCurrentFOC(0);
+    private final StatusSignal<Double>
+            VOLTAGE_STATUS_SIGNAL = motor.getMotorVoltage(),
+            CURRENT_STATUS_SIGNAL = motor.getTorqueCurrent();
```

Comment:
you're not using status signals correctly. They should be stored and frequency updated in the constants